flepimop/gempyor_pkg/src/gempyor/model_info.py

```diff
@@ -203,27 +202,27 @@ def __init__(
                         "modifiers"
                     ].get()
                     raise ValueError(
-                        "Not implemented yet"
+                        "This feature has not been implemented yet."
                     )  # TODO create a Stacked from all
 
             ## NEED TO IMPLEMENT THIS -- CURRENTLY CANNOT USE outcome modifiers
             elif self.outcome_modifiers_scenario is not None:
                 if config["outcome_modifiers"].exists():
                     raise ValueError(
-                        "An outcome modifiers scenario was provided to ModelInfo but no 'outcome_modifiers' sections in config"
+                        "An outcome modifiers scenario was provided to `ModelInfo` but there is no `outcome_modifiers` sections in config."
```

Comment:
```suggestion
                        "An outcome modifiers scenario was provided to `ModelInfo` but there is no 'outcome_modifiers' section in the config."
```
The config is structured such that there can only be one 'outcome_modifiers' section.